changelog/12271.bugfix.md

```diff
@@ -0,0 +1 @@
+Fix `BlockingIOError` when running `rasa cli` on utterances with more than 5KB of text.
```

Comment:
We should be more specific about which CLI command causes this issue (I think it's `rasa shell`, could also be `rasa interactive`), please double-check with @trammell.